tests/acceptance/features/bootstrap/WebUIUsersContext.php

```diff
@@ -290,14 +316,26 @@ public function theDisabledUserTriesToLogin($username, $password) {
 	}
 
 	/**
-	 * @When the administrator deletes user :username using the webUI
+	 * @When the administrator deletes user :username using the webUI and confirms the deletion using the webUI
```

Comment:
I wasn't thinking about repeating ``using the webUI`` but I guess it makes sense, because we have to be clear that the step does both things ``using the webUI``

Reply:
> I wasn't thinking about repeating `using the webUI` but I guess it makes sense, because we have to be clear that the step does both things `using the webUI`

Yah I was trying to make it explicit.